graviti/manager/branch.py

```diff
@@ -12,7 +12,7 @@
 
 from graviti.exception import ResourceNotExistError
 from graviti.manager.commit import ROOT_COMMIT_ID, NamedCommit
-from graviti.manager.lazy import PagingList
+from graviti.manager.lazy import LazyPagingList
```

Comment:
add double quote to the names in the commit message?

Reply:
The commit type should be `style`
The code behavior and speed are both unchanged.